From 2b6dc22181b76a174d4c7947cf62093e914c5e09 Mon Sep 17 00:00:00 2001
From: Zhiyi Zhang <zzhang@codeweavers.com>
Date: Fri, 23 Jan 2026 14:35:01 +0800
Subject: [PATCH] d2d1: Warn in d2d_device_context_set_error().

This is helpful to find out d2d errors when they happen.

(cherry picked from commit b3cd00817a3c53f8aa28c571238f4e84dda47be2)
---
 dlls/d2d1/device.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/d2d1/device.c b/dlls/d2d1/device.c
index 5a3f8794322..463a4d8e411 100644
--- a/dlls/d2d1/device.c
+++ b/dlls/d2d1/device.c
@@ -199,6 +199,7 @@ static void d2d_device_context_draw(struct d2d_device_context *render_target, en
 
 static void d2d_device_context_set_error(struct d2d_device_context *context, HRESULT code)
 {
+    WARN("code %#lx.\n", code);
     context->error.code = code;
     context->error.tag1 = context->drawing_state.tag1;
     context->error.tag2 = context->drawing_state.tag2;
